src/index.js

```diff
@@ -1,32 +1,36 @@
 var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+var API            = 'https://rickandmortyapi.com/api/character/'
+var xhttp          = new XMLHttpRequest()
```

Comment:
No es recomendable dejar tantos espacios, tener un código limpio ayuda a su lectura. 

```suggestion
const xhttp = new XMLHttpRequest()
```